channeldb/channel.go

```diff
@@ -340,10 +340,10 @@ type OpenChannel struct {
 	// target blockchain as specified by the chain hash parameter.
 	FundingOutpoint wire.OutPoint
 
-	// ShortChanID encodes the exact location in the chain in which the
+	// ShortChannelID encodes the exact location in the chain in which the
 	// channel was initially confirmed. This includes: the block height,
 	// transaction index, and the output within the target transaction.
-	ShortChanID lnwire.ShortChannelID
+	ShortChannelID lnwire.ShortChannelID
```

Comment:
definitely an option, i left it this way because we rely on being able to set this value during initialization throughout the codebase. the alternative seems to be adding a `SetShortChanID`?